about.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,700;1,300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
+    <title>About Me Page</title>
+</head>
+<body tabindex="0">
+    <header>
+        <nav>
+            <a href="#home" class="icon" id = "menu-icon">
+                <i class="fa fa-bars"></i>
+            </a>
+            <ul class="menu-desktop">
+                <li><a href="#news">Portoflio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+                <li><a href="https://www.gmail.com"><img src="./icons/Mail.png" alt="mail button Should apper"></a></li>
+            </ul>
+        </nav>
+    </header>
```

Comment:
- [ ] Please make sure the project contains a `Home` page and `About` page in the project design and It's possible to navigate between the pages.